### ISSUE TITLE
Cant change websocket proxy

### ISSUE THREAD

**Issue by TonyGiorgio** [BitcoinDevShop/mutiny-web-poc#292]
Doesn't work for me locally or with testnet.mutinywallet.com

```
save existing network regtest [SettingStringsEditor.tsx:12](https://localhost:3000/data/Dev/BitcoinDevShop/mutiny-web-poc/frontend/src/components/SettingStringsEditor.tsx)
save node manager network regtest [SettingStringsEditor.tsx:13](https://localhost:3000/data/Dev/BitcoinDevShop/mutiny-web-poc/frontend/src/components/SettingStringsEditor.tsx)
```

When I refresh, it reverts back. Doesnt seem like it even changed.

**Comment by TonyGiorgio** [comment]
I tagged the wrong issue.

**Comment by futurepaul** [comment]
pretty sure this was fixed in #304. we were only saving changes to the other fields when the network was changed
